vlasovsolver/cpu_trans_map.cpp

```diff
@@ -674,23 +657,44 @@ void update_remote_mapping_contribution(
       CellID p_ngbr = INVALID_CELLID;
       CellID m_ngbr = INVALID_CELLID;
 
-      for (const auto& [neighbor, dir] : mpiGrid.get_face_neighbors_of(local_cells[c])) {
+      //if(local_cells[c] == 4 || local_cells[c] == 5) {
```

Comment:
this comment looks like it should be removed

Reply:
Indeed. Removed.